source/dotnet/Library/CppConsoleApp/main.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "iostream"
```

Comment:
nitpick I missed earlier:

```suggestion
#include <iostream>
```